### ISSUE TITLE
Sidebar Search UI

### ISSUE THREAD

**Issue by t4k** [coral-erm/coral#321]
In reviewing #183 (Additional Search Fields) I started to question some of the UI aspects of the sidebar search.

The search fields are cumulative (not sure about all of them, but the few I tested with values in each limited the results that way). I think it is a little confusing that the keyword fields _each_ have a GO! button when they are not independent.

Also, this combined with the auto-submit of the dropdown filter is a little strange.

The “New Search” icon and text at the top is unclear. One doesn’t know it is a link until one clicks it.

I don’t have a strong recommendation right now, but we should discuss how to make this better.

**Comment by techsvcslib** [comment]
I agree that the sidebar can be improved from a UX perspective. I think the way in which search works might also bear looking at more closely, but that’s a different thing.

**Comment by rtatterson** [comment]
Enhancement request that could improve the UX experience of Resources module. I think we could prioritize this work.
